### PR TITLE
Bugfix/update

### PR DESCRIPTION
**Pull request by JMilamber**
1.5.11 - 
- Add Golem Hp to Config
- Re-Sort golem progression, with core and nether towers switching places.
    Old: 
    Land->Ocean->Nether->Core->End->Sky

    New:
    Land->Ocean>Core->Nether->End->Sky

1.5.12 -
- Added updated monolith textures. 
- Removed Totem Block

1.5.13 -
- Give chests Explosion Resistance

1.5.14 -
- Fixed bug where Tower would start crumbling when golem was pushed/moved to just outside of the tower (before it would teleport back) 
- Remove Sky-Tower from structure registry completely

1.5.15 -
- Fixed sounds still playing for player when dying and respawning far away
- Fixed title commands not working in single-player without cheats sometimes. 
    Title commands are now sent as if by the server and are more stable.

### CODE REVIEW

File: src/main/java/com/BrassAmber/ba_bt/entity/hostile/golem/BTGolemEntity.java [hunk 1]
```diff
@@ -18,4 +22,10 @@ protected void registerGoals() {
 		
 		this.goalSelector.addGoal(1, new GolemStompAttackGoal(this, 4.0F, 6));
 	}
+
+	/*********************************************************** Properties ********************************************************/
+
```
Comment (Xratedjunior) [inline src/main/java/com/BrassAmber/ba_bt/entity/hostile/golem/BTGolemEntity.java:27]: Feels like we need an 'Override' here to make it clear?

**Review by Xratedjunior — APPROVED** [review]

File: src/main/java/com/BrassAmber/ba_bt/entity/hostile/golem/BTGolemEntity.java (hunk 1 above)
Comment (JMilamber) [reply src/main/java/com/BrassAmber/ba_bt/entity/hostile/golem/BTGolemEntity.java:27]: Done :)

**Comment by JMilamber** [comment]
@Xratedjunior  
If you could test if this fixes the sound bug mentioned in commit logs that would be awesome, a little short on time atm doing the 1.18.1 update. 
I only tested for the golem/destroy tower one mentioned.

**Comment by Xratedjunior** [comment]
You mean that crash when killing the golem?

**Comment by JMilamber** [comment]
No, that one hasn't given an actual data on when/why/what happened. 

On the Curseforge someone commented and notified me that when the golem left the radius of the tower (but not far enough to be teleported back) that the tower crumble would start. 
This was because of how I was checking to see if the golem was still alive. Ai was using getNearestEntity which works fine until I remembered all of the mobs that spawn just below the golem floor. So I changed how it checks for the golem and it works correctly now. 

The user also mentioned that if you kill the golem, but then die and respawn far away you continue to hear the sound effects for a bit. 

I changed the function calling the sound event to level.playLocalSound but didn't get to test if the change fixes the problem.

**Pull request merged**